.github/workflows/ci.yaml

```
@@ -34,6 +34,45 @@ jobs:
       - run: npm ci
       - run: npm run compile-test
       - run: npm test
+  scarb-pick-test:
```

```suggestion
  test-ui-scarb
```

let's be consistent in naming